pkg/ocm/clusters.go

```diff
@@ -112,6 +112,9 @@ type Spec struct {
 	// HyperShift options:
 	Hypershift     Hypershift
 	BillingAccount string
+
+	// Machine pool's storage
+	MachinePoolRootDisk *cmv1.AWSVolumeBuilder
```

Comment:
I think we do not want to expose the sdk type in this struct, and instead keep this struct as native types. We then convert to the builder funtions later on when needed.

Reply:
Can you clarify? Are you suggesting a `String` type instead of the SDK or another Struct that is not part of the SDK? Thanks

Reply:
Ok thanks, changed.